code/datums/components/space_kidnap.dm

```diff
@@ -39,7 +39,7 @@
 	var/obj/particles = new /obj/effect/abstract/particle_holder (victim, /particles/void_kidnap)
 	kidnapping = TRUE
 
-	if(do_after(parent, kidnap_time, victim, extra_checks = victim.incapacitated))
+	if(do_after(parent, kidnap_time, victim, extra_checks = CALLBACK(victim, PROC_REF(check_incapacitated))))
```

Comment:
```suggestion
	if(do_after(parent, kidnap_time, victim, extra_checks = CALLBACK(src, PROC_REF(check_incapacitated), victim))))
```

Caught in 4k not testing code

Reply:
![image](https://github.com/user-attachments/assets/c629d685-6965-4726-98ce-a098809f9840)